src/views/Timesheets/components/SavedShiftsList/SavedShiftsList.scss

```diff
@@ -0,0 +1,19 @@
+@import '../../../../vars';
+
+// Make rows alternate white and grey
+/* stylelint-disable */
+.shift-list > .shift-item:nth-child(even) {
+  background: #ebeaea;
+  &:hover {
+    background-color: $primary-cyan;
+    //   cursor: pointer;
```

Comment:
Is there a plan for the cursor: pointer lines here and on 17?

Reply:
Thanks for picking that up, we will remove those